### PR TITLE
Created article page for Inkscape tool

### PR DESCRIPTION
**Pull request by moingshaikh**
Created a page for Inkscape tool under Designers-tools directory.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/designer-tools/inkscape/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Inkscape is an open-source vector-based graphic editing tool used by graphic designers, artists andillustrationist to create high quality,
+vector-based illustration and designs. Inkscape is widely termed as 'free and open-source' alternative 
+to popular illustration design tool Adobe Illustrator.
+
+Inkscape can be used to create or edit vector based graphics including illustrations, shapes, line arts, charts,logos, diagrams, 
+as well as raster images.
+
+Being an open-source tool, Inkscape comes free of cost. 
+
+You can freely download Inkscape [here](https://inkscape.org/release/0.92.3/)
```
Comment (jonathan-grah) [inline guide/english/designer-tools/inkscape/index.md]: This line can be removed

File: guide/english/designer-tools/inkscape/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: Inkscape
+---
+
+## Inkscape
+
+Inkscape is an open-source vector-based graphic editing tool used by graphic designers, artists andillustrationist to create high quality,
+vector-based illustration and designs. Inkscape is widely termed as 'free and open-source' alternative 
+to popular illustration design tool Adobe Illustrator.
+
+Inkscape can be used to create or edit vector based graphics including illustrations, shapes, line arts, charts,logos, diagrams,
```
Comment (jonathan-grah) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
Inkscape can be used to create or edit vector-based graphics including illustrations, shapes, line arts, charts,logos, diagrams, 
```

File: guide/english/designer-tools/inkscape/index.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Inkscape can be used to create or edit vector based graphics including illustrations, shapes, line arts, charts,logos, diagrams, 
+as well as raster images.
+
+Being an open-source tool, Inkscape comes free of cost. 
+
+You can freely download Inkscape [here](https://inkscape.org/release/0.92.3/)
+
+It is available for all major Operating Systems including Windows, Linux, and Mac OS X.
+
+Download link for particular Operating Systems can be found here:
```
Comment (jonathan-grah) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
Download links for particular Operating Systems can be found here:
```

File: guide/english/designer-tools/inkscape/index.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+to popular illustration design tool Adobe Illustrator.
+
+Inkscape can be used to create or edit vector based graphics including illustrations, shapes, line arts, charts,logos, diagrams, 
+as well as raster images.
+
+Being an open-source tool, Inkscape comes free of cost. 
+
+You can freely download Inkscape [here](https://inkscape.org/release/0.92.3/)
+
+It is available for all major Operating Systems including Windows, Linux, and Mac OS X.
```
Comment (jonathan-grah) [inline guide/english/designer-tools/inkscape/index.md]: Tag this on to the previous line

**Review by jonathan-grah — CHANGES_REQUESTED** [review]

**Comment by moinshaikh** [comment]
Recommended changes done. If you or someone can take a look and advise if anything else needs to be done.

Thanks!

File: guide/english/designer-tools/inkscape/index.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+to popular illustration design tool Adobe Illustrator.
+
+Inkscape can be used to create or edit vector-based graphics including illustrations, shapes, line arts, charts,logos, diagrams, 
+as well as raster images.
+
+Being an open-source tool, Inkscape comes free of cost. 
+
+It is available for all major Operating Systems including Windows, Linux, and Mac OS X.
+
+Download links for particular Operating Systems can be found here:
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]: If you could link to download page, that's be great, versions keep changing

File: guide/english/designer-tools/inkscape/index.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+- [Windows](https://inkscape.org/release/0.92.3/windows/)
+- [GNU/Linux](https://inkscape.org/release/0.92.3/gnulinux/)
+- [Mac OS X](https://inkscape.org/release/0.92.3/mac-os-x/dl/)
+
+You can also get access to the raw source code of Inskcape [here](https://inkscape.org/release/0.92.3/source/)
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]: Same goes for this

File: guide/english/designer-tools/inkscape/index.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+Further Information and Tutorials Resources:
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
### More Information:
```

File: guide/english/designer-tools/inkscape/index.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+---
+title: Inkscape
+---
+
+## Inkscape
+
+Inkscape is an open-source vector-based graphic editing tool used by graphic designers, artists andillustrationist to create high quality,
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
Inkscape is an open-source vector-based graphic editing tool used by graphic designers, artists andillustrationist to create high quality, vector-based illustration and designs. Inkscape is widely termed as 'free and open-source' alternative to popular illustration design tool Adobe Illustrator.
```

File: guide/english/designer-tools/inkscape/index.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+vector-based illustration and designs. Inkscape is widely termed as 'free and open-source' alternative
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
```

File: guide/english/designer-tools/inkscape/index.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
+---
+title: Inkscape
+---
+
+## Inkscape
+
+Inkscape is an open-source vector-based graphic editing tool used by graphic designers, artists andillustrationist to create high quality,
+vector-based illustration and designs. Inkscape is widely termed as 'free and open-source' alternative 
+to popular illustration design tool Adobe Illustrator.
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
```

File: guide/english/designer-tools/inkscape/index.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+Inkscape can be used to create or edit vector-based graphics including illustrations, shapes, line arts, charts,logos, diagrams, 
+as well as raster images.
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
```

File: guide/english/designer-tools/inkscape/index.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
...
+title: Inkscape
+---
+
+## Inkscape
+
+Inkscape is an open-source vector-based graphic editing tool used by graphic designers, artists andillustrationist to create high quality,
+vector-based illustration and designs. Inkscape is widely termed as 'free and open-source' alternative 
+to popular illustration design tool Adobe Illustrator.
+
+Inkscape can be used to create or edit vector-based graphics including illustrations, shapes, line arts, charts,logos, diagrams,
```
Comment (thecodingaviator) [inline guide/english/designer-tools/inkscape/index.md]:
```suggestion
Inkscape can be used to create or edit vector-based graphics including illustrations, shapes, line arts, charts,logos, diagrams, as well as raster images.
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Check comments

**Comment by thecodingaviator** [comment]
@jonathan-grah If your issue have been cleared, please go ahead and approve the PR. If you no longer want to take it up, please remove your assignment too. Thanks :)

**Review by jonathan-grah — APPROVED** [review]

**Comment by jonathan-grah** [comment]
@thecodingaviator, thanks yeah, much better idea to not refer to version numbers in links, I agree 👍

**Comment by RandellDawson** [comment]
@jonathan-grah Are you approving the PR as is or are you expecting the OP to make the changes requested by @thecodingaviator?

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**